### PR TITLE
I323/parsing bv x variables

### PR DESCRIPTION
**Pull request by mikand**
Fixed issue #323 by allowing bvX variables in the parser.

### CODE REVIEW

**Comment by marcogario** [comment]
Memo: Tests on z3 back conversion using smtlib need to be reenabled for bv (didn't check yet)

**Comment by mikand** [comment]
@marcogario The back conversion on bv has been re-enabled (see https://github.com/pysmt/pysmt/pull/326/commits/223ea43cd316a51f28edf0feb840ab44475f5d23) and it is the issue breaking the tests...

**Comment by marcogario** [comment]
Ok, assign it to me if you want me to look into this.

File: pysmt/test/test_regressions.py [hunk 1]
```diff
@@ -289,6 +289,16 @@ def test_parse_define_fun_bind(self):
...
 
+    def test_parse_bvx_var(self):
+        smtlib_input = """
+        (declare-fun bv1 () (_ BitVec 8))
+        (assert (bvult (_ bv0 8) (bvudiv (bvmul (bvadd bv1 (_ bv1 8)) (_ bv5 8)) (_ bv5 8))))
+        (check-sat)"""
+        parser = SmtLibParser()
+        buffer_ = cStringIO(smtlib_input)
+        parser.get_script(buffer_)
+
```
Comment (marcogario) [inline pysmt/test/test_regressions.py]: This is only testing that the code does not crash. We can do much better, and actually test that we obtain the expression defined above.

**Pull request merged**